templates/citizen_project/_form.html.twig

```diff
@@ -92,6 +92,10 @@
             'placeholder': 'Un court paragraphe sur la ou les solutions que vous proposez',
         }
     }) }}
+    <small>
+        Ex. Le projet a pour objectif de mettre en relation ces élèves avec un réseau de professionnels
```

Comment:
c'est pas plutôt "**s**es" ?

Reply:
Mmmm alors là j ai même pas cherché à voir si il n y avait des fautes :D étant pas un as de l'orthographe :D . J ai juste copié collé par rapport au ticket.

Reply:
Can you please confirm @guillaumedenhez or @marineb ?

Reply:
it's ok guys "ces" pour désigner les élèves du dessus